static/app/bootstrap/initializeSdk.tsx

```diff
@@ -109,4 +110,6 @@ export function initializeSdk(config: Config, {routes}: {routes?: Function} = {}
     Sentry.setTag('sentry_version', window.__SENTRY__VERSION);
   }
   Sentry.setTag('rrweb.active', hasReplays ? 'yes' : 'no');
+
+  LongTaskObserver.startPerformanceObserver();
```

Comment:
I think you could move this later in the lifecycle of the app if we used the buffered flag. It's not a bit deal where we init though, but we should probably still use the buffered flag to try and catch the long tasks from before the observer is initialized

Reply:
We can, I was pretty sure last time I tried it (a long time ago) it wasn't working, just going to do this for now and we can add buffered in the future if it seems valuable (for us it'll likely just catch first render which you can already guess will be a long task)